container-hardening.md

```diff
@@ -0,0 +1,33 @@
+# Container Hardening
+
+In general, container hardening is the proceess of utilizing container scanning tools to detect possible CVEs (Common Vulnerabilities & Exposures) and remediating them to minimize the attack surface of a container. Each individual, organization, or other entity may form their own requirements for this process and its outputs based on their needs and risk acceptance. Below is an example:
+
+> We are building a web server container using [Apache HTTP](https://httpd.apache.org/) version 2.4.48 as the server. Once built, we analyzed the container with a scanning tool (Anchore, Twistlock, Snyk, etc.) and detected [CVE-2021-39275](https://nvd.nist.gov/vuln/detail/CVE-2021-39275) was present. After reviewing the CVE, we decided to upgrade the Apache HTTP server to the newer version 2.4.51 in order to prevent the vulnerability from being exploited and remove the threat from the container.
+
+## SDP and Containter Hardening
+
```

Comment:
some additional sections might be helpful. 

1. A Repository Structure section that explains the layout of the repository would be good.  i.e., how each directory is a container and each directory has a Dockerfile and Makefile and whatever else.
2. A "How-To" section might be helpful.   just a quick list of commands you use to build images and build the tars and what not.

Reply:
added a new issue to update the README (#36 ) for the first item